contracts/GNTPaymentChannels.sol

```diff
@@ -1,149 +0,0 @@
-pragma solidity ^0.5.3;
```

Comment:
Why is this file removed?

Reply:
it was not used afaik, can put it back if it still has value :)

Reply:
For me it is ok to remove it, but I would double check someone else.